borgmatic/hooks/mysql.py

```diff
@@ -86,6 +86,10 @@ def execute_dump_command(
     mysql_dump_command = tuple(
         shlex.quote(part) for part in shlex.split(database.get('mysql_dump_command') or 'mysqldump')
     )
+
+    dump_format = database.get('format')
```

Comment:
Style nit: This variable could easily inlined given that it's not used anywhere else besides the next line.